recipe/bld.bat

```diff
@@ -1,4 +1,23 @@
 robocopy . "%PREFIX%" /S /XF bld.bat
 if errorlevel NEQ 1 exit 1
 
+cd ..
+if errorlevel 1 exit 1
+
+%PYTHON% "%RECIPE_DIR%\get_plugins.py"
+if errorlevel 1 exit 1
+
+copy "UAC\Plugins\x86-unicode\UAC.dll " "%PREFIX%\Plugins\x86-unicode\"
+if errorlevel 1 exit 1
+copy "Untgz\untgz\unicode\untgz.dll" "%PREFIX%\Plugins\x86-unicode\"
+if errorlevel 1 exit 1
+copy "UnicodePathTest_1.0\Plugin\UnicodePathTest.dll" "%PREFIX%\Plugins\x86-unicode\"
+if errorlevel 1 exit 1
+
+copy "%SRC_DIR%\COPYING" + "UAC\License.txt" + "Untgz\untgz\LICENSE.TXT" "LICENSE.txt"
+if errorlevel 1 exit 1
```

Comment:
Looks like this rams the license files together without breathing room (e.g. blank lines). Not totally sure how to fix this on Windows, but it seems like it should be fixable.